pulseguardian/templates/users_listing.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'base.html' %}
+
+{% block body %}
+<div id="message_info" class="col-md-12">
```

Comment:
The view handler isn't ever passing any messages into the template, so leave this out. We can add it in later if we ever need it.